### ISSUE TITLE
Labels

### ISSUE THREAD

**Issue by sanjithpk** [kranfix/flutter_radar_chart#1]
![image](https://user-images.githubusercontent.com/40496139/71719408-48aa5e00-2e44-11ea-98a8-a9f089ce9080.png)
Is there any way to label the corners (AA BB CC) like in the figure in your generated chart?

**Comment by RaistlinTAO** [comment]
I second this, the labels are crucial

**Comment by kranfix** [comment]
Hello, sorry for late response. Yes, it is posible, but not automatically:  
 

```dart
RadarChart(

  length: 8,

  radius: 82,

  radialColor: Colors.gry,

  radialStroke: 0.5,

  radars: [

    RadarTile(...),

    RadarTile(...),

  ],

  vertices: [

    for (int i = 0; i < competenciesInfo.length; i++)

     RadarVertex(

        textOffset: textOffsets[i]

        .... 
     ),

  ],

);
```

Where the `textOffsets` are predefined, but it could be calculated by a function considering initial `RadarChart.initialAngle`:

```dart
const List<Offset> radarOffsets = [
  Offset(15, -10),
  Offset(15, -5),
  Offset(-42, 15),
  Offset(-100, -10),
  Offset(-90, -15),
  Offset(-95, -30),
  Offset(-35, -40),
  Offset(5, -30),
];
```

  
And the `RadarVertex` is an PreferredSizedWidget and it would be centered in a vertex:

```dart
class RadarVertex2 extends StatelessWidget with PreferredSizeWidget {
  final double radius;
  final Widget text;
  final Offset textOffset;

  RadarVertex2({
    this.radius,
    this.text,
    this.textOffset,
  });

  @override
  Size get preferredSize => Size.fromRadius(radius);

  @override
  Widget build(BuildContext context) {
    Widget tree = CircleAvatar(
      radius: radius,
      backgroundColor: Colors.green,
    );
    if (text != null) {
      tree = Stack(
        children: [
          tree,
          if (textOffset == null)
            text
          else
            Transform.translate(
              offset: textOffset,
              child: text,
            )
        ],
      );
    }
    return tree;
  }
}
```

**Comment by daybson** [comment]
The origin/center is changing to another place...

```
List<double> values1 = [2 / 5, 3 / 5, 4 / 5, 3 / 5, 1 / 5];

RadarChart(
                            length: 5,
                            radius: 100,
                            initialAngle: 0,
                            backgroundColor: Colors.white,
                            borderStroke: 2,
                            borderColor: Colors.grey.shade300,
                            radialStroke: 1,
                            radialColor: Colors.grey.shade300,
                            vertices: [
                              RadarVertex(
                                radius: radius,
                                textOffset: Offset(-50, -50), text: Text('0'),
                              ),
                              for (int i = 0; i < values1.length; i++)
                                RadarVertex(
                                  radius: radius,
                                  text: Text('F'),
                                  textOffset: Offset(0, 0),
                                ),
                            ],
                            radars: [
                              RadarTile(
                                radialColor: Colors.red,
                                values: values1,
                                borderStroke: 2,
                                borderColor: Colors.yellow,
                                backgroundColor: Colors.yellow.withOpacity(0.4),
                              ),
                            ],
                          ),
```


![Screenshot_1633969970](https://user-images.githubusercontent.com/3179898/136824792-612f1721-c973-450c-ba7b-804fc406f4d5.png)

**Comment by kranfix** [comment]
Oh, let me reproduce this. Thanks, in advance.

**Comment by kranfix** [comment]
Solved with #6 .  
Published: https://pub.dev/packages/radar_chart/versions/2.1.0
